solution/backend/resources/admin.py

```diff
@@ -398,13 +414,66 @@ def save(self, commit=True):
 
 @admin.register(SupplementalContent)
 class SupplementalContentAdmin(AbstractResourceAdmin):
+    change_list_template = "admin/import_resources_button.html"
     form = SupContentForm
     list_display = ("date", "name", "description", "category", "updated_at", "approved", "name_sort")
     list_display_links = ("date", "name", "description", "category", "updated_at")
     search_fields = ["date", "name", "description"]
     fields = ("approved", "name", "description", "date", "url", "category",
               "locations", "bulk_title", "bulk_locations", "internal_notes", "location_history")
 
+    def get_urls(self):
+        urls = super().get_urls()
+        my_urls = [
+            path('import_resources/', self.show_import_resources_page),
+        ]
+        return my_urls + urls
+
+    def show_import_resources_page(self, request):
+        error = None
+
+        if "import" in request.POST:
+            try:
+                return render(request, "admin/resources_imported.html", context={
+                    "num_conversions": 1,
+                    "conversions": None,
+                })
+            except ValidationError as e:
+                error = e.message
+            except Exception as e:
+                error = str(e)
+        if "get_template" in request.POST:
```

Comment:
does this do anything?

Reply:
wasnt ready for review, was still a draft, forgot to mark it as such